featurex/converters/api.py

```diff
@@ -1,12 +1,21 @@
 import os
-from featurex.stimuli.text import ComplexTextStim
+import base64
+import json
+from featurex.stimuli.text import DynamicTextStim, ComplexTextStim
 from featurex.converters.audio import AudioToTextConverter
 
 try:
     import speech_recognition as sr
```

Comment:
Even though it will increase redundancy slightly, we should probably import `sr` inside the initializers; that way we don't have to worry about catching exceptions.